community-operators/metering/meteringoperator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -328,7 +445,7 @@ spec:
                     value: "30"
                   - name: RELEASE_HISTORY_LIMIT
                     value: "3"
-                  image: quay.io/coreos/metering-helm-operator:0.13.0
+                  image: quay.io/openshift/origin-metering-helm-operator:latest
```

Comment:
Should this be using a tagged image?

Reply:
Ideally yes, but I'm not sure whether we need to be using 4.0 or 4.1 and if the 4.1 images are being mirrored out regularly or not yet.

Reply:
Yeah, currently 4.1 tags aren't available, so I guess we will use 4.0 until 4.1 is mirrored out.